README.md

```diff
@@ -224,9 +224,13 @@ Attributes are returned as an instance of `com.yoti.api.client.Attribute<T>`.  S
 `com.yoti.api.client.Anchor` represents how a given `Attribute<T>` has been _sourced_ or _verified_.  These values are created and signed whenever a Profile Attribute is created, or verified with an external party.
 
 For example, an attribute value that was _sourced_ from a Passport might have the following values:
-| anchor.getType | SOURCE |
-| anchor.getValue | PASSPORT |
-| anchor.getTimestamp | 2017-10-31, 19:45:59.123789 |
+
+`Anchor` property | Example value
+-----|------
+anchor.getType | SOURCE
+anchor.getValue | PASSPORT
+anchor.getSubType | OCR
+anchor.getSignedTimestamp | 2017-10-31, 19:45:59.123789
```

Comment:
Should this be anchor.getSignedTimeStamp.getTimestamp?

Reply:
Well spotted, thank you.  Yes it should.  I'll amend the commit.

Reply:
Actually, I'll change it to use field names and drop the method references.  It looks neater that way, and it's trivial for the reader to work out the getters they should use.